charts/super-agent/templates/install-job.yaml

```diff
@@ -0,0 +1,49 @@
+{{- if .Values.helm.create -}}
```

Comment:
I would add some docs either here or in the readme explaining why we have a job

Reply:
Done on [#5157357](https://github.com/newrelic/helm-charts/pull/1175/commits/51573576d221c3e4ea5e8ba0278965e4860cd5ed)